Podverse/Constants.swift

```diff
@@ -288,28 +288,37 @@ enum EpisodeActions: String {
     }
 }
 
-enum ErrorMessages: String {
-    case noClipsAvailable
-    case noClipsInternet
-    case noPlaylistsAvailable
-    case noPlaylistsInternet
-    case noPlaylistsNotLoggedIn
+struct Strings {
     
-    var text: String {
-        get {
-            switch self {
-            case .noClipsAvailable:
-                return "No clips available"
-            case .noClipsInternet:
-                return "No internet connection"
-            case .noPlaylistsAvailable:
-                return "No playlists available"
-            case .noPlaylistsInternet:
-                return "No internet connection"
-            case .noPlaylistsNotLoggedIn:
-                return "Login to view your playlists"
-            }
-        }
+    struct Errors {
+        static let noClipsAvailable = "No clips available"
+        static let noClipsInternet = "No internet connection"
+        static let noDownloadedEpisodesAvailable = "No downloaded episodes available"
+        static let noEpisodeClipsAvailable = "No clips available for this episode"
+        static let noEpisodesAvailable = "No episodes available"
+        static let noPlaylistsAvailable = "No playlists available"
+        static let noPlaylistsInternet = "No internet connection"
+        static let noPlaylistsNotLoggedIn = "Login to view your playlists"
     }
     
 }
+
+func checkForConnectivity() -> Bool {
```

Comment:
Probably not a good place to have this function. This file is a constants file. I suggest you make a Utilities.swift file to have these kind of methods in. Super simple, non class specific, general methods

Reply:
Should Utilities.swift be a class? Then I would call the functions from other files with something like Utilities.shared.checkForConnectivity? Or something else?...

Reply:
I don't think it needs to be a class. These to functions are not using and kind of `self` reference so they can just be used like in the constants file but in a different one called Utilities.swift. Only because Constants should only contain constants :)